README.md

```diff
@@ -1,118 +1,80 @@
+![Basic Hejbit Swarm setup in External Storage in Nextcloud administration](https://github.com/user-attachments/assets/47e0cdda-5c1a-4464-a752-cf1eb5eeb19d)
 
-### Using Swarm
+### Usage
 
-Once setup and configured, the Bee node is ready to use.
+Once setup and configured, the Decentralized Storage is ready to use.
 
--   Navigate to "Files" option on the menu -> "External Storage" and then the name of the Bee node.
--   Click to Upload file(s):
+- Access the Decentralized Storage folder under "All Files" or via the "Files" menu under "External Storage."
+- Upload files as you would in Nextcloud.
 
-![Upload file](assets/images/swarm_Files_UploadFile.png "Upload file")
+![Upload file to swarm in Nextcloud](https://github.com/user-attachments/assets/aadd664e-26ca-470a-a27b-af8d94351e52)
 
-<em>Upload file to swarm in Nextcloud</em>
+- Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
 
--   Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
+![View, download, and file view actions in Nextcloud](https://github.com/user-attachments/assets/596b72ed-d97f-48ba-bcb9-0ee5ff581a3c)
 
-![View file and action menu](assets/images/swarm_Files_ViewFile.png "View file and action menu")
+- Files from another Nextcloud storage can also be copied/moved to the decentralized storage.. Choose the decentralized storage folder as the target Folder
 
-<em>View, download, and file view actions in Nextcloud</em>
+![Copying or moving files to decentralized storage folder](https://github.com/user-attachments/assets/ceed3585-f7e6-4f16-b371-d61402e9f1e9)
 
--   Files from another Nextcloud storage can also be copied/moved to the Bee node. Choose the Bee node as the target Folder
+- Right-click on a Swarm file to copy the Swarm reference (hash) to your clipboard. Alternatively, click the three dots in the Actions menu and select 'Copy Swarm Reference.' The Swarm reference is the unique address of the file on the Swarm network.
 
-![Copy file](assets/images/swarm_CopyMove.png "Copy file")
+![Copy swarm reference to clipboard](https://github.com/user-attachments/assets/cc73282b-e32e-411f-a94b-a2ac3313f60b)
 
--   Right-click on a swarm file to copy the swarm reference (or hash) to the clipboard. Alternatively click the 3 dots of the Actions menu and select the menu option "Copy swarm reference". The swarm reference is the unique address of the file on the swarm.
+### Get Hejbit License
 
-![Copy swarm reference to clipboard](assets/images/swarm_Files_CopySwarmRef.png)
+Experience the future of data storage with 5GB of free, decentralized storage on the Ethereum Swarm network. Our plugin seamlessly integrates with your existing Nextcloud interface, providing true data sovereignty without the complexity.
 
-<em>Copy swarm reference to clipboard</em>
+[Get your free licence here!](https://metaprovide.org/hejbit/start) .
```

Comment:
@frozenbanana The point after the link was only a fix to prevent opening the thumbnail on collectives. You can remove it from here.